src/rsz/test/buffer_varying_lengths.ok

```diff
@@ -1,27 +1,285 @@
-[INFO RSZ-0059] Removed 71 buffers.
-[INFO RSZ-0051] Resized 1500 instances: 1500 up, 0 up match, 0 down, 0 VT
-[WARNING RSZ-0062] Unable to repair all setup violations.
-[INFO RSZ-0033] No hold violations found.
-worst slack max -0.302
-tns max -30.979
-No differences found.
+[INFO RSZ-0034] Found 51 slew violations.
+[INFO RSZ-0036] Found 56 capacitance violations.
+[INFO RSZ-0038] Inserted 111 buffers in 56 nets.
+   Iter   |    Area   | Removed | Inserted |   Pins
+          |           | Buffers | Buffers  | Remaining
+-------------------------------------------------------
+        0 |     +0.0% |       0 |        0 |       100
+       10 |    +11.3% |      30 |       40 |        90
+       20 |    +39.0% |      50 |       76 |        80
+       30 |    +62.6% |      68 |      109 |        70
+       40 |    +71.2% |      86 |      136 |        60
+       50 |    +87.5% |      99 |      162 |        50
+       60 |   +104.8% |     108 |      185 |        40
+       70 |   +124.6% |     108 |      203 |        30
+       80 |   +138.4% |     108 |      218 |        20
+       90 |   +146.9% |     108 |      227 |        10
+    final |   +152.3% |     111 |      238 |         0
+-------------------------------------------------------
+
+Solution under tight timing constraint:
+Group                                  Slack
+--------------------------------------------
+clk                                     1.04
+clk                                    -0.35
+
+max slew
+
+Pin                                    Limit    Slew   Slack
+------------------------------------------------------------
+i1-59/Z                                 0.20    0.06    0.14 (MET)
+
+max capacitance
+
+Pin                                    Limit     Cap   Slack
+------------------------------------------------------------
+i1-59/Z                                60.65   28.39   32.26 (MET)
+
+-------------------------------------------------------------
+ Distance | Buffer area | Path delay | Sequence
+-------------------------------------------------------------
+     25.0 |        0.00 |       0.00 | 
+     50.0 |        0.00 |       0.00 | 
+     75.0 |        0.00 |       0.00 | 
+    100.0 |        0.00 |       0.00 | 
+    125.0 |        0.00 |       0.00 | 
+    150.0 |        0.00 |       0.00 | 
+    175.0 |        0.00 |       0.00 | 
+    200.0 |        0.00 |       0.00 | 
+    225.0 |        0.00 |       0.01 | 
+    250.0 |        0.00 |       0.01 | 
+    275.0 |        0.00 |       0.01 | 
+    300.0 |        1.86 |       0.03 |  BUF_X4
+    325.0 |        1.86 |       0.04 |  BUF_X4
+    350.0 |        1.86 |       0.04 |  BUF_X4
+    375.0 |        1.86 |       0.04 |  BUF_X4
+    400.0 |        1.86 |       0.04 |  BUF_X4
+    425.0 |        1.86 |       0.04 |  BUF_X4
+    450.0 |        1.86 |       0.04 |  BUF_X4
+    475.0 |        1.86 |       0.05 |  BUF_X4
+    500.0 |        1.86 |       0.05 |  BUF_X4
```

Comment:
This is worse on both timing and area than the corresponding result from the second table:
```
    500.0 |        0.00 |       0.02 | 
 ```
The delay is measured driver pin to load pin, if we fix it to include gate delay that will probaly show that there actually was timing benefit to inserting the X4 buffer.